docs/inference-providers/index.md

```diff
@@ -144,71 +149,381 @@ completion = client.chat.completions.create(
+
+Install with NPM:
+
+```shell
+npm install @huggingface/inference
+```
+
+Then use the client with Javascript:
+
+```js
+import { InferenceClient } from "@huggingface/inference";
+
+const client = new InferenceClient(process.env.HF_TOKEN);
+
+const chatCompletion = await client.chatCompletion({
+  model: "deepseek-ai/DeepSeek-V3-0324",
+  messages: [
+    {
+      role: "user",
+      content: "How many 'G's in 'huggingface'?",
+    },
+  ],
+});
+
+console.log(chatCompletion.choices[0].message);
+```
+
+</hfoption>
+
+<hfoption id="openai">
+
+If you're already using OpenAI's Python client, then you need a **drop-in OpenAI replacement**. Just swap-out the base URL to instantly access hundreds of additional open-weights models through our provider network.
+
+```javascript
+import OpenAI from "openai";
+
+const client = new OpenAI({
+  baseURL: "https://router.huggingface.co/v1",
+  apiKey: process.env.HF_TOKEN,
+});
+
+const completion = await client.chat.completions.create({
+  model: "deepseek-ai/DeepSeek-V3-0324",
+  messages: [
+    {
+      role: "user",
+      content: "How many 'G's in 'huggingface'?",
+    },
+  ],
+});
+
+console.log(completion.choices[0].message.content);
+```
+
+</hfoption>
+
+<hfoption id="fetch">
+
+**Native Fetch Integration**: For lightweight applications or custom implementations, use our REST API directly with standard fetch:
 
 ```js
 import fetch from "node-fetch";
 
 const response = await fetch(
-    "https://router.huggingface.co/novita/v3/openai/chat/completions",
-    {
-        method: "POST",
-        headers: {
-            Authorization: `Bearer ${process.env.HF_TOKEN}`,
-            "Content-Type": "application/json",
+  "https://router.huggingface.co/v1/chat/completions",
+  {
+    method: "POST",
+    headers: {
+      Authorization: `Bearer ${process.env.HF_TOKEN}`,
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      model: "deepseek-ai/DeepSeek-V3-0324",
+      messages: [
+        {
+          role: "user",
+          content: "How many 'G's in 'huggingface'?",
         },
-        body: JSON.stringify({
-            provider: "novita",
-            model: "deepseek-ai/DeepSeek-V3-0324",
-            messages: [
-                {
-                    role: "user",
-                    content: "How many 'G's in 'huggingface'?",
-                },
-            ],
-        }),
-    }
+      ],
+    }),
+  }
 );
 console.log(await response.json());
 ```
 
-For convenience, the JS library `@huggingface/inference` provides an [`InferenceClient`](https://huggingface.co/docs/huggingface.js/inference/classes/InferenceClient) that handles inference for you. You can install it with `npm install @huggingface/inference`.
+</hfoption>
+
+</hfoptions>
+
+#### HTTP / cURL
+
+For testing, debugging, or integrating with any HTTP client, here's the raw REST API format. Our intelligent routing automatically selects the optimal provider for your requested model:
```

Comment:
This is not true, right? Routing is based on user settings preference (at least for now?)